data-serving/data-service/Dockerfile

```diff
@@ -23,5 +26,25 @@ RUN npm run-script build
 # Expose service on port 3000.
 EXPOSE 3000
 
-# Start service.
-CMD [ "npm", "start" ]
\ No newline at end of file
+# Specify a command for up to this step.
+CMD [ "npm", "start" ]
+
+# Multi-staged build, we don't need a full node image to run the app.
+FROM node:alpine as prod
```

Comment:
What happens when we don't specify a version here? Does it just use the max version number?

https://hub.docker.com/_/node/

Reply:
Ah I had forgotten the version indeed, done now, used 12 to match the build version.